account_receipt_send/README.rst

```diff
@@ -0,0 +1,84 @@
+=======================
+Receipts Print and Send
+=======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--invoicing-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-invoicing/tree/15.0/account_receipt_send
+    :alt: OCA/account-invoicing
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-invoicing-15-0/account-invoicing-15-0-account_receipt_send
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/webui/builds.html?repo=OCA/account-invoicing&target_branch=15.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+Adds a button to Send and Print Receipts
+Adds a new mail template for receipts
```

Comment:
- [ ] Add configuration section also mandatory in all modules (products)

https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/CONFIGURE.rst

- Go to Invoicing -> Configuration and activate -> Customer Invoices -> "Sale Receipt"
![imagen](https://user-images.githubusercontent.com/8736623/221908318-bc4f2a2c-bf7a-47cf-9a6d-e36f5cb552c8.png)


This is USAGE:

- Go to Invoicing -> Customers - > Receipts -> Create a new receipt
- Select a customer and a producto and confirm
- Clic in send button (actually doesnt works so there is a BUG)